aspnetcore/tutorials/razor-pages/da1.md

```diff
@@ -0,0 +1,125 @@
+---
+title: Updating the generated pages
+author: rick-anderson
+description: Updating the generated pages with better display.
+keywords: ASP.NET Core, Razor Pages
+ms.author: riande
+manager: wpickett
+ms.date: 08/07/2017
+ms.topic: get-started-article
+ms.technology: aspnet
+ms.prod: asp.net-core
+uid: tutorials/razor-pages/da1
+---
+
+# Updating the generated pages
+
+By [Rick Anderson](https://twitter.com/RickAndMSFT)
+
+We have a good start to the movie app, but the presentation is not ideal. We don't want to see the time (12:00:00 AM in the image below) and **ReleaseDate** should be **Release Date** (two words).
+
+![Movie application open in Chrome showing movie data](sql/_static/m55.png)
+
+## Update the generated code
+
+Open the *Models/Movie.cs* file and add the highlighted lines shown in the following code:
+
+[!code-csharp[Main](razor-pages-start/sample/RazorPagesMovie/Models/MovieDate.cs?name=snippet_1&highlight=10-11)]
+
+Right click on a red squiggly line **> Quick Actions and Refactorings**.
```

Comment:
- Add a comma between "red" and "squiggly".
- Remove the bold styling from the ">" character.